component/events.md

```diff
@@ -50,6 +50,205 @@ Components have the ability to bind to special [can.events.inserted inserted] an
         }
       }
 
+## Listening to viewModel events
+
+As in [can.Control] we can listening for events inside or outside the Component. We can use a Templated Event Handler for this. Unlike [can.Control] the value inside `{Name}` are looked up on `viewModel` first followed by `this.options` and then the `window` object.
+You can use the events for:
+* Tracking changes on [can.List] or [can.Map] like `add`, `remove`, `change`, (`replaced`)
+* Tracking changes on properties on the viewModel
+* Tracking changes on [can.Model]-Instance like `created`, `updated`, `destroyed`
+* Tracking promises on [can.Deferred] 
+
+The following example shows all the described possibilities:
```

Comment:
I should restate ... it's not so much about what people can handle (although I think that a 200 line example is a bit long to read)

... it's about having the right docs in the right places, especially API docs, and being able to maintain docs.  `Control` and `Component.prototype.events` are just "sugar" ontop of `can.Map.prototype.bind` and `can.List.prototype.bind`.  Those `.bind` functions should be the place where comprehensive docs about what events are possible are documented.  This makes sense because if we make changes to `.bind`, we will likely document it there.

The `Component.prototype.events` should just be explaining how the sugar maps to the underlying `.bind`.